### PR TITLE
Remove POLARION_TEST_SUITE_PARAMS

### PR DESCRIPTION
**Pull request by qinqon**
**Is this a BUG FIX or a FEATURE ?**:

> Uncomment only one, leave it on its own line:
>
> /kind bug
> /kind enhancement

**What this PR does / why we need it**:
Remove POLARION_TEST_SUITE_PARAMS, the test at tier1 are going to use one parmeter so skipping spaces is not needed, also when https://github.com/kubevirt/qe-tools/pull/21 is merged tier1 will be run with multiple parameters separated by commas.
```
./test/e2e/openshift.cnv.workflow.sh \
    --polarion-custom-plannedin=2_4 \
    --polarion-execution=true \
    --polarion-project-id=CNV \
    --polarion-report-file=polarion_results.xml \
    --test-suite-params=env_tier=tier1
```

**Special notes for your reviewer**:

**Release note**:

```release-note

```

### CODE REVIEW

**Comment by qinqon** [comment]
@phoracek they are ok just passing one of the parameters while the PR is merged at qe-tools, Nelly is the only maintainer there that's why is slow but she knows about it.

**Comment by phoracek** [comment]
I'm not sure I understand.

Is this change needed now (before the one on qe-tools gets in)?

Is it dependent in any way on the qe-tools PR?

`It will not needed when kubevirt/qe-tools#21 is merged` suggests that the qe-tools PR must be merged before this one. If it is not the case, I miss the real motivation in the commit message.

**Comment by qinqon** [comment]
@phoracek let me rephrase the message, is all wrong.
